maintenance_tasks.gemspec

```diff
@@ -19,9 +19,11 @@ Gem::Specification.new do |spec|
   spec.bindir = "exe"
   spec.executables = ["maintenance_tasks"]
 
-  spec.post_install_message = "Thank you for installing Maintenance Tasks "\
-    "#{spec.version}. To complete, please run:\n\nrails generate "\
-    "maintenance_tasks:install"
+  spec.post_install_message = <<~MESSAGE.strip
```

Comment:
Can we remove this post install message? It show every time the gem is installed, even if it is not in the first time and the generator was already installed, so it is more confusing than helpful.